src/databricks/labs/ucx/workspace_access/listing.py

```diff
@@ -44,7 +46,7 @@ def _progress_report(self, _):
             )
 
     def _list_workspace(self, path: str) -> Iterable[ObjectInfo]:
-        list_retry_on_value_error = retried(on=[InternalError], timeout=self._verify_timeout)
+        list_retry_on_value_error = retried(on=retryable_exceptions, timeout=self._verify_timeout)
```

Comment:
why do we need to retry on more errors than InternalError? this is already taking 18 hours to run.